File: app/src/main/java/com/example/review/Setting.java
package com.example.review;

import android.content.Context;
import android.content.SharedPreferences;

public class Setting {

    static public SharedPreferences        sp;
    static public SharedPreferences.Editor edit;

    public static void init(Context context) {
        sp = context.getSharedPreferences("ActivitysData", Context.MODE_PRIVATE);
        edit = sp.edit();
    }

    public static boolean getBoolean(String extra) {
        return sp.getBoolean(extra, false);
    }

    public static int getInt(String extra) {
        return sp.getInt(extra, 0);
    }

    public static void set(String extra, boolean value) {
        edit.putBoolean(extra, value).commit();
    }

    public static void set(String extra, int value) {
        edit.putInt(extra, value).commit();
    }
}

File: app/src/main/java/com/example/review/Setting.kt
package com.example.review

import android.content.Context
import android.content.SharedPreferences
import android.content.SharedPreferences.Editor

object Setting {
    var sp: SharedPreferences? = null
    var edit: Editor? = null

    fun init(context: Context) {
        sp = context.getSharedPreferences("ActivitysData", Context.MODE_PRIVATE)
        edit = sp!!.edit()
    }

    fun getBoolean(extra: String?): Boolean {
        return sp!!.getBoolean(extra, false)
    }

    fun getInt(extra: String?): Int {
        return sp!!.getInt(extra, 0)
    }

    operator fun get(extra: String?): Boolean {
        return sp!!.getBoolean(extra, false)
    }

    operator fun set(extra: String?, value: Boolean) {
        edit!!.putBoolean(extra, value).commit()
    }

    operator fun set(extra: String?, value: Int) {
        edit!!.putInt(extra, value).commit()
    }
}